release_docs/RELEASE.txt

```diff
@@ -204,16 +274,6 @@ New Features
         H5_FCTESTDLL(VAR)        --> H5FC_TEST_DLL(VAR)
         HDF5_HL_F90CSTUBDLL(VAR) --> H5FC_HL_DLL(VAR)
 
-    - The ROS3 VFD now includes the HTTP status code and a generic message
```

Comment:
While this work did occur, the libcurl backend is removed with this PR, so this entry is no longer relevant.